app/src/main/java/edu/uc/groupproject/covid19tracker/service/CasesByCountryDataProvider.kt

```diff
@@ -35,11 +35,11 @@ class CasesByCountryDataProvider {
                      * data.
                      */
                     val casesByCountry = response.body()!!.string()
-                    val Jobject = JSONObject(casesByCountry)
-                    val Jarray = Jobject.getJSONArray("countries_stat")
-                    for (i in 0 until Jarray.length()) {
-                        val `object` = Jarray.getJSONObject(i).getString(dataType)
-                        data.add(`object`)
+                    val jObject = JSONObject(casesByCountry)
+                    val jArray = jObject.getJSONArray("countries_stat")
+                    for (i in 0 until jArray.length()) {
+                        val objString = jArray.getJSONObject(i).getString(dataType)
+                        data.add(objString)
```

Comment:
Renamed variables to align with coding standards.  It should use camel case notation.

Reply:
I'd take this one step further.  Having a JSONObject variable named jObject is like having a String variable named "string".  Change this to a name that better represents its contents.  And yes, make it lowerCamelCase.